spec/features/bidder_interacts_with_auction_spec.rb

```diff
@@ -85,4 +113,83 @@
     expect(page).to have_content("No bids yet.")
   end
 
+  scenario "Viewing bid history for running auction" do
+    Timecop.scale(36000) do
+      create_running_auction
+    end
+    path = auction_bids_path(@auction.id)
+    visit path
+
+    # sort the bids so that newest is first
```

Comment:
what about removing this comments? These methods are pretty small/clean for reading without comments. Alternatively, if you wanted to make this all super easy to read, you could wrap these bits of code in private methods. 

so it would be `bids = newest_first(@action.bids)`

and then at the bottom of this spec file:

```ruby
def newest_first(bids)
  bids.sort_by {|bid| bid.created_at}.reverse
end
```